pysiaf/tests/test_nirspec.py

```diff
@@ -149,6 +155,13 @@ def test_nirspec_aperture_transforms(verbose=False):
 
         # aperture
         aperture = siaf[aper_name]
+        # offset slightly from default tilt values
```

Comment:
Do I understand correctly that this is to add the tilt case to the tests? 
If yes, I suggest to add another loop in there which makes sure that both `tilt=None` and `tilt=something` are tested.
In the current implementation it seems like only the `tilt=something` case is exercised.